src/TMap.cpp

```diff
@@ -2584,3 +2585,11 @@ void TMap::update()
     }
 }
 
+QString TMapLabel::base64EncodePixmap()
```

Comment:
:thinking: You do not need to use the `QString` as an intermediate here - the `QByteArray` that `QByteArray::toBase64()` outputs that is perfectly fine to use - all the way to converting it back to a form that can be squirted into the Lua subsystem (with a `lua_pushstring(L, bytearray.constData());`)...